### ISSUE TITLE
Regarding amount of data used for Make-Model classification

### ISSUE THREAD

**Issue by koutilya-pnvr** [Charleo85/DeepCar#4]
Hi, 

In the ReadMe, it was mentioned that you were able to get an accuracy of 93.2% (top-5) for make-model classification. I am wondering how much data you used. Is it all of the 136k images or the data given as part of classification_train.txt and classification_test.txt (Just 431 makes-models pairs out of 1716)? 

Also is the deep_car.py the main code for the make-model classification? Because in the .ipynb notebook file, you only seem to consider a make-model pair only if it has atleast 125 images (3rd box in the .ipynb file, if num < 125: continue). That is not the case i the deep_car.py python file though. This is a bit confusing. 

Will be great if you can clarify both. Thanks in advance.
.

**Comment by koutilya-pnvr** [comment]
Also is the reported 93.2% accuracy on validation dataset or on the training dataset?

**Comment by JUSTDODoDo** [comment]
hello,I am very interested in your work on RA-CNN Implemented  by pytorch, but as a newbie, how do I prepare to train the dataset, I have already got the entire dataset of Compcars, you have not explained clearly in Readme. Training, can you tell the details, thank you!
